hydra_base/lib/groups.py

```diff
@@ -53,11 +53,14 @@ def add_resourcegroup(group, network_id,**kwargs):
     db.DBSession.flush()
     return group_i
 
-def delete_resourcegroup(group_id,**kwargs):
+def delete_resourcegroup(group_id, purge_data='N', **kwargs):
     """
         Add a new group to a scenario.
     """
     group_i = _get_group(group_id)
+
+    if purge_data == 'Y':
+        network._purge_datasets_unique_to_resource('GROUP', group_id)
```

Comment:
Has this got anything to do with scenario inheritance?

Reply:
no, it was just a fix I noticed :)